### PR TITLE
Fix redirection after signup via code; signup email text fix

### LINKED ISSUE #167: Signin via new email yields Sign-up-like email but doesn't create session
I found that if I try to sign-in via unregistered email I still get an account but I don't get a session when I enter the code.
I think this is a bug.

How to reproduce:
- create a new project via  `bb -e '(load-string (slurp "https://biffweb.com/new-project.clj"))'`
- `bb dev`
- Go go `localhost:8080/signin` (not signup but signin)
- enter an email
- enter code
- Result is HTTP 303 to `http://localhost:8080/signin?error=not-signed-in`

So I didn't get a session to sign in with and it created the user.

The email link has SignUp and SignIn header and texts:

```
TO: new@aaaa.com
SUBJECT: Sign up for My Application

We received a request to sign in to My Application using this email address. Enter the following code to sign in:

981163

This code will expire in three minutes. If you did not request this code, you can ignore this email.
```

So the subject is "sign UP" and the content is "sign IN".

So one of these have to be changed to either redirect into `/app` or the UI has to crash and say that this user doesn't exist :thinking_face:

### PR DESCRIPTION
**Pull request by Invertisment**
Closes #167

### CODE REVIEW

File: src/com/biffweb/impl/auth.clj [hunk 1]
```diff
@@ -226,7 +226,7 @@
       {:status 303
        :headers {"location" app-path}
        :session (assoc session :uid (or existing-user-id
-                                        (get-user-id db email)))}
+                                        (get-user-id (xt/db db) email)))}
```
Comment (Invertisment) [inline src/com/biffweb/impl/auth.clj:229]: The newly created user doesn't exist in this `db` so it's needed to find from the newest version.

Comment (jacobobryant) [reply src/com/biffweb/impl/auth.clj]: This throws an exception; it should be `(xt/db node)`. I just pushed the fix.

**Comment by jacobobryant** [comment]
Thanks for catching this!

File: src/com/biffweb/impl/auth.clj (hunk 1 above)
Comment (Invertisment) [reply src/com/biffweb/impl/auth.clj]: Ah. Right. I didn't test this code but I wanted exactly what you said.

**Pull request merged**